projects/1.18/assets/electrodynamics/electrodynamics/lang/zh_cn.json

```
@@ -1 +1,555 @@
+  "subtitles.electrodynamics.rodhittingground" : "轨道炮弹丸：击中地面",
+  "subtitles.electrodynamics.latherunning" : "车床：运行",
+  "subtitles.electrodynamics.equipheavyarmor" : "复合装甲防具：铿锵",
+  "subtitles.electrodynamics.hydroelectricgenerator" : "水轮旋转",
+  "subtitles.electrodynamics.seismicscanner" : "地质扫描仪：运行",
+  "subtitles.electrodynamics.electrolyticseparator" : "电解分离机：运行",
+  "subtitles.electrodynamics.nightvisiongoggleson" : "夜视护目镜：启动",
+  "subtitles.electrodynamics.nightvisiongogglesoff" : "夜视护目镜：关闭",
+  "subtitles.electrodynamics.jetpackswitchmode" : "喷气背包：切换模式",
+  "subtitles.electrodynamics.jetpack" : "喷气背包：引擎运行",
+  "keycategory.electrodynamics" : "电动力学",
+  "key.electrodynamics.jetpackascend" : "喷气背包喷气",
+  "key.electrodynamics.togglenvgs" : "开启/关闭夜视护目镜",
+  "key.electrodynamics.jetpackmode" : "切换喷气背包模式",
+  "key.electrodynamics.servoleggingsmode" : "切换紧身裤模式",
+  "key.electrodynamics.toggleservoleggings" : "开启/关闭紧身裤特殊功能",
+
+  "jei.info.item.compositearmor" : "这一套重型盔甲使玩家得以承受爆炸的伤害。然而，玩家必须穿戴完整套装且保证胸甲中已安装陶瓷板。穿戴复合装甲胸甲时，手持陶瓷板右击以安装。复合装甲胸甲最多能同时装载两片陶瓷板。",
+  "jei.info.item.upgradeiteminput" : "自动抽取升级是自动化狂热者最好的伙伴之一。应用该升级的方块会从用户所指定方向的临近方块中抽取物品。该升级提供两种模式：默认模式与智能模式。默认模式下，被抽取的物品会试图根据用户所指定的方向顺序来填补所有的输入槽位。在方向指定中重复指定同一个方向没有意义。智能模式下，每个输入槽按顺序被映射到一个对应的方向。例如，假设一台机器有2个输入槽，而该升级中指定的方向是\"北\"和\"南\"。以这个顺序，输入槽1会抽取来源于北方的方块的物品，而输入槽2会抽取来源于南方的方块的物品。你也可以多次指定相同的抽取方向，如\"北\" \"北\"。",
+  "jei.info.item.upgradeitemoutput" : "自动弹出升级是自动化狂热者最好的伙伴之一。应用该升级的方块会向用户所指定方向的临近方块弹出物品。该升级提供两种模式：默认模式与智能模式。默认模式下，该升级会尝试提取本机器中所有的输出槽（包括输出槽2），并将被提取的物品根据用户所指定的方向顺序，向相应方向弹出。在方向指定中重复指定同一个方向没有意义。智能模式下，每个输出槽按顺序被映射到一个对应的方向。例如，假设一台机器有2个输出槽，而该升级中指定的方向是\"北\"和\"南\"。以这个顺序，输出槽1会向北方的方块弹出物品，而输出槽2会向南方的方块弹出物品。你也可以多次指定相同的弹出方向，如\"北\" \"北\"。",
+  "jei.info.item.railgunplasma" : "伤害：\n    初始：40\n    1秒后:20",
+  "jei.info.item.railgunkinetic" : "接受的弹药：\n    钢棒\n    不锈钢棒\n    高强度低合金钢棒\n    初始伤害：\n    钢棒：16\n    不锈钢棒：20\n    高强度低合金钢棒：4\n\n使用方法：\n一只手持动能轨道炮，另一只手持金属棒。",
+  "jei.info.item.coalgeneratorfuelsource" : "燃煤发电机燃料：\n    剩余燃烧时间：%s秒",
+  "jei.info.block.coalgenerator" : "输出：\n    最大电压：120V\n    最大功率：4.08 kW\n    最大电流：34.0 A",
+  "jei.info.block.upgradetransformer" : "特点：\n    99.25%的转换效率\n    输出电压是输入电压的两倍\n    无输入限制",
+  "jei.info.block.downgradetransformer" : "特点：\n    99.25%的转换效率\n    输出电压是输入电压的二分之一\n    无输入限制",
+  "jei.info.block.solarpanel" : "输出：\n    最大电压：120V\n    最大功率：1.2 kW\n    最大电流：10.0 A",
+  "jei.info.block.advancedsolarpanel" : "输出：\n    最大电压：240V\n    最大功率：4.08 kW\n    最大电流：20.0 A",
+  "jei.info.block.thermoelectricgenerator" : "输出：\n    最大电压：120V\n    最大功率：500 W\n    最大电流：4.12 A\n    \n    热源：熔岩",
+  "jei.info.block.combustionchamber" : "输出：\n    最大电压：120V\n    最大功率：7.0 kW\n    最大电流：2.2 A\n    \n    燃料来源\n    乙醇\n    氢气",
+  "jei.info.block.hydroelectricgenerator" : "输出：\n    最大电压：120V\n    最大功率：720 W\n    最大电流：6 A",
+  "jei.info.block.windmill" : "输出：\n    最大电压：120V\n    最大功率：1.2 kW\n    最大电流：10.0 A\n\n理想高度：Y=135",
+  "jei.info.block.tank" : "最多能储存%s mB的任何流体。注意：流体储罐被破坏时流体也会消失！",
```

```suggestion
  "jei.info.block.tank" : "最多能储存%smB的任何流体。注意：流体储罐被破坏时流体也会消失！",
```
https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/pull/1563#discussion_r811929684